config/environments/production.rb

```diff
@@ -95,7 +95,11 @@
   end
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.delivery_method = :smtp
+  if ENV['USE_SENDMAIL'] == 'true'
+    config.action_mailer.delivery_method = :sendmail
```

Comment:
I would prefer something like
```ruby
     config.action_mailer.delivery_method = ENV.fetch('SMTP_ELIVERY_METHOD', 'smtp').to_sym
```

What do you think?

Reply:
@dsander sorry I lost track of this PR! Sounds great. I'll push up an amended patch.